esmvaltool/cmorizers/obs/cmorize_obs_duveiller2018.py

```diff
@@ -0,0 +1,150 @@
+"""ESMValTool CMORizer for Duveiller2018 data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   https://ndownloader.figshare.com/files/9969496
+
+Last access
+   20190430
+
+Download and processing instructions
+   - Download the dataset albedo_IGBPgen.nc and save in the right directory
+     according to ESMValTool practices.
+   - Complete the CMOR-config specifications (see instructions in the file
+     itself)
+   - Run cmorize_obs
+
+Modification history
+   20190430-crezee_bas: written based on cmorize_obs_Landschuetzer2016.py.
+
+Caveats
+   Please be aware that the selected vegetation transition code is not written
+   to the filename, since this would break the ESMValTool file naming
+   conventions.
+"""
+
+import datetime
+import logging
+import os
+from warnings import catch_warnings, filterwarnings
+
+import cf_units
+import iris
+import numpy as np
+
+
+from .utilities import (fix_coords, fix_var_metadata, save_variable,
+                        set_global_atts)
+
+logger = logging.getLogger(__name__)
+
+
+def fix_time_coord_duveiller2018(cube):
+    """Fix the time coordinate for dataset Duveiller2018."""
+    # Rename 'Month' to 'time'
+    cube.coord('Month').rename('time')
+
+    # Create arrays for storing datetime objects
+    custom_time = np.zeros((12), dtype=object)
+    custom_time_bounds = np.empty((12, 2), dtype=object)
+    custom_time_units = 'days since 1950-01-01'
```

Comment:
Is no time needed? e.g. `"days since 1850-01-01 00:00:00.0"`

Reply:
That is definitely safer, so I will include it :)